src/python/grpcio_tests/tests/unit/_compression_test.py

```diff
@@ -13,37 +13,98 @@
 # limitations under the License.
 """Tests server and client side compression."""
 
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
 import unittest
 
+import contextlib
+from concurrent import futures
 import logging
+import os
+
 import grpc
+import grpc.compression
 from grpc import _grpcio_metadata
 
 from tests.unit import test_common
 from tests.unit.framework.common import test_constants
+from tests.unit import _tcp_proxy
+
+os.environ['GRPC_EXPERIMENTAL_DISABLE_FLOW_CONTROL'] = 'true'
 
 _UNARY_UNARY = '/test/UnaryUnary'
+_UNARY_STREAM = '/test/UnaryStream'
+_STREAM_UNARY = '/test/StreamUnary'
 _STREAM_STREAM = '/test/StreamStream'
 
+# Cut down on test time.
+_STREAM_LENGTH = test_constants.STREAM_LENGTH // 2
+
+
+def make_handle_unary_unary(pre_response_callback):
```

Comment:
Do you think we should make these handler creating functions private to this module?

Reply:
Agree in principle, but in practice, this test module is a top-level application. Since it will never be imported, is it wholly necessary to make its members private?